test/components/views/voip/PipView-test.tsx

```diff
@@ -306,6 +316,12 @@ describe("PipView", () => {
             setUpRoomViewStore();
             viewRoom(room.roomId);
             mkVoiceBroadcast(room);
+            await actFlushPromises();
+
+            if (!voiceBroadcastPlaybacksStore.getCurrent()) {
+                throw new Error("broadcast not started");
```

Comment:
Should we use `expect` here with `toBeDefined()` ?

Reply:
Will move it to the test :+1: